modular_skyrat/modules/modular_implants/code/nifs.dm

```diff
@@ -255,6 +256,73 @@
 		if(nifsoft.type == nifsoft_to_find)
 			return nifsoft
 
+///NIFSoft Remover. This is mostly here so that security and antags have a way to remove NIFSofts from someome
```

Comment:
make sure you add a traitor-themed child or except it from being used for illegal tech when you add it to the uplink, if you plan to.